cmp/internal/value/format.go

```diff
@@ -0,0 +1,258 @@
+// Copyright 2017, The Go Authors. All rights reserved.
```

Comment:
Unfortunately, it seems that git does not have a concept of "copy this file with history".

https://stackoverflow.com/questions/16937359/git-copy-file-preserving-history